3303Elevator/src/app/Config/local.properties

```diff
@@ -0,0 +1,10 @@
+time.multiplier=0
+
+scheduler.address=localhost
+scheduler.port=3000
```

Comment:
scheduler needs a port for elevatorComms, and floor comms

And then Logger needs an address with 3 ports. One each to receive from floor, scheduler,elevator